com.github.alexr4535.siglo.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id" : "com.github.alexr4535.siglo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "siglo",
+    "finish-args" : [
+        "--allow=bluetooth",
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=system-bus",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "gatt",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "pip3 install --prefix=/app --no-cache-dir gatt"
+            ]
+        },
+        {
+            "name": "dbus-python",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "pip3 install --prefix=/app --no-cache-dir dbus-python"
+            ]
+        },
+        {
+            "name": "python3-requests",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "pip3 install --prefix=/app --no-cache-dir requests"
+            ]
+        },
+        {
+            "name": "python3-pyxdg",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
```

Comment:
This (and above) won't work, there is no network.

For the Python dependencies, please have a look at https://github.com/flatpak/flatpak-builder-tools to generate a manifest for python dependencies..

Reply:
thanks, yeah i understand this is still outstanding issue.

Reply:
done